src/zope/app/apidoc/typemodule/type.py

```diff
@@ -69,10 +69,11 @@ def items(self):
         """See zope.container.interfaces.IReadContainer"""
         results = [(name, LocationProxy(iface, self, name))
                    for name, iface in getUtilitiesFor(self.interface)]
-        results.sort(lambda x, y: cmp(x[1].getName(), y[1].getName()))
+        results.sort(key=lambda x: x[1].__name__)
```

Comment:
Perhaps LocationProxy is not working correctly, then, because I was getting `AttributeError` about `getName`. I will look again.